1-exercises/J-string-substring/exercise3.js

```diff
@@ -8,7 +8,8 @@
 
 let statement = "I do not like programming";
 
-let result = "";
+// let result = "I do not like programming".replace("not","");
+let result=statement.substring(0,4)+statement.substring(8);
 
```

Comment:
Correct. Just watch your whitespace. Check that you format your code with Prettier when you're done. This is more readable code:
```javascript
let result = statement.substring(0, 4) + statement.substring(8);
```